src/services/types.ts

```diff
@@ -589,7 +603,7 @@ namespace ts {
 
     export interface ClassifiedSpan {
         textSpan: TextSpan;
-        classificationType: ClassificationTypeNames;
+        classificationType: ClassificationTypeNames | number;

```

Comment:
As discussed earlier this needs to be enum and we need to make sure that ClassifiedSpan with `classificationType: ClassificationTypeNames | number;` is only if format is supplied otherwise it is `ClassificationTypeNames` for backward compatibility of the api